tests/ut_mdesktopentry.cpp

```diff
@@ -396,6 +395,7 @@ void UtMDesktopEntry::localization()
 
     if (!lang.isEmpty()) {
         qputenv("LANG", lang.toLocal8Bit());
+        qputenv("LC_ALL", lang.toLocal8Bit());
```

Comment:
It was enough for the old code (that did the `$LANG` parsing manually), but with glib, I had failing tests with `LC_ALL=de_AT.UTF-8` from the client environment.